### PR TITLE
Adds dynamic displaycard generation.

### LINKED ISSUE #113: Create dynamic content on the display card page.
Once issue #40 is resolved, the display card can use parameterized URLs in order to dynamically add the accounts for that card to the web page.

### PR DESCRIPTION
**Pull request by paytonshaltis**
DON'T MERGE UNTIL AFTER CODING MILESTONE 2!

Closes #113. Introduces a small bug that will be fixed after #149 is merged.

Dynamic card content can now be seen for a specific Goon Card! Scanning the QR Code and navigating to the generated dynamic card now shows the first and last name of the owner, as well as the accounts that they selected to be shared for that card. 

NOTE: This PR introduces a small bug. If you try deleting an account that is associated with some generated Goon Card, the display card page may show 'undefined undefined' for the user's name. This bug is fixed in #149 by implementing the proper cascade code, meaning that deleting any `user_account` entry will delete all corresponding `card_entry` entries, so be mindful of this while testing.

### CODE REVIEW

File: views/notfound/index.ejs [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <title>Not Found</title>
   </head>
   <body id="body" class="h-100 text-center text-white dark-mode bg-dark">
+    <% var user = undefined; %>
```
Comment (sderacy) [inline views/notfound/index.ejs:10]: Oh I noticed you did what I did in #154 😂

**Review by sderacy — APPROVED** [review]
Works like a charm. Nice job!!

File: db/cardEntry.js [hunk 2]
```diff
@@ -46,7 +46,66 @@ const getCardEntries = async (uuid) => {
...
+    return null;
+  }
+};
+
+const getUserData = async (uuid) => {
+  // Find just a user_account ID associated with the given uuid.
+  let result = await knex("card_entries")
+    .select("user_account_id")
+    .where("uuid", uuid)
+    .limit(1)
```
[the remark this answers is not in the document]
Comment (paytonshaltis) [reply db/cardEntry.js:81]: This is part of figuring out the first and last name of the owner of the card. Given the UUID, all we know are the accounts that became associated with it. So, by getting just one of these accounts, we can look at who the owner of that account is. Getting _all_ of the accounts serves no use here, so I specify to get at most 1.

**Comment by thadakn1** [comment]
just ran it, incredible work @paytonshaltis !! worked perfectly on my end! 👏🏼

**Pull request merged**